src/test/kotlin/org/jetbrains/research/transformations/util/ParserSetup.kt

```diff
@@ -9,7 +9,8 @@ import java.util.logging.Logger
  */
 object ParserSetup {
     private val LOG = Logger.getLogger(javaClass.name)
-    private const val TARGET_PATH = "/tmp/pythonparser"
+    private const val PARSER_NAME = "pythonparser.py"
+    private val TARGET_PATH = "${System.getProperty("java.io.tmpdir")}$PARSER_NAME"
```

Comment:
Thank you for this improvement! But should we add "/" between ("java.io.tmpdir")} and $PARSER_NAME?
${System.getProperty("java.io.tmpdir")}/$PARSER_NAME"

Reply:
I am not sure, because I get this path with a slash

Reply:
hm, and I have this exception:
`Exception in thread "main" java.io.FileNotFoundException: /tmppythonparser (Permission denied)`

Reply:
Hm, I will check if a slash exists and add it if does not